recipe/build.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+export NOCONFIGURE=1
```

Comment:
This is interesting.  Why is it set to noconfigure?

Reply:
Please add a comment

Reply:
> `./autogen.sh` will run `configure` unless you tell it not too... Alternatively we could pass all the configure flags to `./autogen.sh` which will pass them to the configure script.

Reply:
Comment has been added